util/minetestmapper.py

```diff
@@ -151,19 +154,16 @@ def usage():
 path = None
 output = "map.png"
 border = 0
-scalecolor = "black"
-bgcolor = "white"
+scalecolor = "white"
```

Comment:
Ok, I reverted those changes. Those were related to my usage, but now I set color in my program instead.